### DISCUSSION TITLE
Can I plot another candlestick chart below the original one?

### DISCUSSION THREAD

**Discussion by hz7451** [kernc/backtesting.py#549] [Q&A] [answered]
I want to plot two candlestick charts, can I pass OHLC data to self.I and plot it? Thank you

**Comment by kernc** [comment, marked as answer]
[`Strategy.I()`](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Strategy.I) only supports overlaying or appending line and scatter plots, but [`Backtest.plot()`](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Backtest.plot) does return you a [Bokeh Figure](https://docs.bokeh.org/en/latest/docs/reference/plotting/figure.html#bokeh.plotting.Figure) object and maybe you can fiddle with it yourself:
https://github.com/kernc/backtesting.py/blob/267d99f0d48745acf98156b2e69150c02d42f761/backtesting/backtesting.py#L1592
https://github.com/kernc/backtesting.py/blob/267d99f0d48745acf98156b2e69150c02d42f761/backtesting/_plotting.py#L718-L727
